From 74b653fb34c9ff65f80eca016ff0188ec5bb95c1 Mon Sep 17 00:00:00 2001
From: Timple Raj M <quic_timple@quicinc.com>
Date: Mon, 12 Jun 2023 16:48:29 +0530
Subject: [PATCH] arm64: defconfig: Enable QCOM M31 and UNIPHY USB phy driver

Enable QCOM M31 and UNIPHY USB phy driver present in IPQ5332

Change-Id: I790eaa321ab8882b1b3bdef1ca139321c00fb42c
Signed-off-by: Varadarajan Narayanan <quic_varada@quicinc.com>
Signed-off-by: Timple Raj M <quic_timple@quicinc.com>
---
 arch/arm64/configs/defconfig | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/arch/arm64/configs/defconfig b/arch/arm64/configs/defconfig
index 16bb111789f9..1d07073506eb 100644
--- a/arch/arm64/configs/defconfig
+++ b/arch/arm64/configs/defconfig
@@ -1240,6 +1240,8 @@ CONFIG_PHY_QCOM_USB_HS=m
 CONFIG_PHY_QCOM_USB_SNPS_FEMTO_V2=m
 CONFIG_PHY_QCOM_USB_HS_28NM=m
 CONFIG_PHY_QCOM_USB_SS=m
+CONFIG_PHY_QCOM_M31_USB=y
+CONFIG_PHY_IPQ_UNIPHY_USB=y
 CONFIG_PHY_RCAR_GEN3_PCIE=y
 CONFIG_PHY_RCAR_GEN3_USB2=y
 CONFIG_PHY_RCAR_GEN3_USB3=m
-- 
2.34.1

